banzai/main.py

```diff
@@ -222,9 +222,12 @@ def parse_directory_args(runtime_context=None, raw_path=None, extra_console_argu
 
 def reduce_directory(runtime_context=None, raw_path=None, image_types=None):
     # TODO: Remove image_types once reduce_night is not needed
+    pipeline_context, raw_path = parse_directory_args(runtime_context, raw_path)
```

Comment:
Why pipeline context and runtime context?

Reply:
Oops, this line was a remnant from the merge, I've removed it